articles/relnotes/index.md

```diff
@@ -449,7 +450,7 @@ Please note that the operation RandomWalkPhaseEstimation from the namespace Micr
 ### Known Issues
 
 - The `--filter` option to `dotnet test` does not work correctly for tests written in Q#.
-  As a result, individual unit tests cannot be run in Visual Studio Code; we recommend using `dotnet test` at the command line to re-run all tests.
+  Individual unit tests can be run by filtering by Trait, or use `dotnet test` at the command line to run all tests.
```

Comment:
This is confusing: The first sentence tells me the filtering feature of dotnet test doesn't work. The second sentence seems to contradict this by saying I _can_ do filtering with dotnet test.

Reply:
Right. What would be the correct command to filter by unit test name (the corresponding trait) on the command line with the new setup?

Reply:
I added tags to the generated code for a trait named "Target". So you should be able to use the --filter option against Targets: `dotnet test --filter "Target=QuantumSimulator"`. [link] (https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-test?tabs=netcore21#filter-option-details) I would definitely double check the exact syntax.

Reply:
In addition to the Target trait, they also have an Operation trait that will have the value of the operation's unqualified name.

Reply:
Looking closer, actually it looks like I shouldn't have changed that file, since the listed issues are specific to particular versions. I'll hence revert that back to its original state, and then we can add more detail on filtering with your blog post.